site/src/pages/components/invokers.explainer.mdx

```diff
@@ -180,17 +180,17 @@ event type will be `'interest'`. If the element has _Shown Interest_, and
 interest moves away from both the _Interest Element_ and the _Interestee_, then
 the element _Loses Interest_ and an `InterestEvent` with the type of
 `'loseinterest'` will be dispatched on the _Interestee_. The event also contains
-a `relatedTarget` property that will reference the _Interest_ element.
+a `invoker` property that will reference the _Interest_ element.
 `InterestEvents` are always non-bubbling, cancellable events.
 
 ```webidl
 [Exposed=Window]
 interface InterestEvent : Event {
   constructor(DOMString type, InterestEventInit interestEventInit);
-  readonly attribute Element relatedTarget;
+  readonly attribute Element invoker;
 };
 dictionary InterestEventInit : EventInit {
-  Element relatedTarget;
+  Element invoker;
```

Comment:
Probably fine but are we okay with Invoker in an interest event?

Reply:
On one hand I can see why we'd not want to, but on the other I think having the symmetry helps understand the concept of _what this thing is_. Additionally there's no good word for "thing that interest was shown on" - it's lacking in the explainer.

We could change `invoker` to `invokeTarget` and then have a matching `interestTarget` which more closely matches the IDL, but these are the _opposite_ of what the IDL properties are (`invokeTargetElement` is the `event.target`, while the thing with `invokeTargetElement` becomes `event.invoker`).

All this to say I think `invoker` for both is the least-worst option.

Reply:
I think I agree I certainly think it *shouldn't* match the IDL and for lack of a better term Invoker will do.